dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -321,8 +333,14 @@ def main(
     worker_class,
     pre_import,
     multiprocessing_method,
+    get_cluster_conf,
     **kwargs,
 ):
+    if get_cluster_conf:
+        client = Client(scheduler)
+        print_cluster_config(client)
+        sys.exit(0)
+
```

Comment:
This needs to move further down (and respect `scheduler_file` and `security` arguments).